x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/shared_fields/snapshot_policies_field.tsx

```diff
@@ -194,8 +194,8 @@ export const SnapshotPoliciesField: React.FunctionComponent = () => {
                 }
                 euiFieldProps={{
                   'data-test-subj': 'snapshotPolicyCombobox',
+                  fullWidth: false,
                   options: policies,
-                  singleSelection: { asPlainText: true },
```

Comment:
I think `singleSelection` prop is still needed as policy name otherwise looks a bit different (not like a text and suggesting that multiple values are possible)

Reply:
right, this was definitely an accident 😅